### DISCUSSION TITLE
post/python-environments/

### DISCUSSION THREAD

**Discussion by giscus** [basnijholt/nijho.lt#15] [General]
# post/python-environments/

A Personal Journey Through Pyenv, Pipenv, venv, Virtualenv, Homebrew, Anaconda, Miniconda, and finally Micromamba

https://www.nijho.lt/post/python-environments/

**Comment by black-ether** [comment]
micromamba is looks great but it's  toooooo looooong to type! they need to change it to something like mb.

**Reply by basnijholt** [reply]
I agree! I alias it to `mm`:

https://github.com/basnijholt/.dotbins/blob/d3aee9afc5bbbe7a285dafd2853f9916420cfc00/shell/zsh.sh#L51

**Reply by black-ether** [reply]
wow, i don't know you can do that! awesome!
